src/power_menu.rs

```diff
@@ -33,32 +33,53 @@ pub struct CliArgs {
     /// Show the menu items and exit
     #[arg(short, long)]
     pub list_items: bool,
+
+    /// Switch to elogind
+    #[arg(short, long, default_value = "false")]
```

Comment:
why not just define it as `SessionManager`, e.g.

```rust
    #[arg(short, long)]
    pub session_manager: SessionManager,
```

and

```rust
#[derive(ValueEnum, Debug, Clone, Default)]
pub enum SessionManager {
    Elogind,
    #[default]
    Systemd,
}
```

Reply:
Oh yeah, that does seem more elegant. Done